renovate.json

```diff
@@ -88,6 +89,72 @@
       ],
       "enabled": false,
       "description": "The version of the ZAC Docker image in our Docker Compose file is set to 'latest' and is only used for local development"
+    },
+    {
```

Comment:
Renovate has quite a few grouping rules pre-specified. I'd suggest to see how useful these are in our setup and figure out how to tune it from there. 

Some other settings seem incorrect here, I guess you can just remove all these added PackageRules for now.

Reply:
What about the rules defined for dependabot? Will we have to re-create those for renovate? Or do that after we get a request to update those dependencies (ie. angular, node, bootstrap) ?

Reply:
Better to do that afterwards. Let's first see what grouping it comes up with automatically and we can easily create new packagerules based on that

Reply:
Agreed. One the reasons we want to migrate to Renovate in the first place is that it is much more clever and has a lot of pre-defined rules. In my fork where I tested out Renovate some time ago I did not have to create any rules I think.